web/text_layer_builder.js

```diff
@@ -79,22 +81,23 @@ var TextLayerBuilder = (function TextLayerBuilderClosure() {
      *   for specified amount of ms.
      */
     render: function TextLayerBuilder_render(timeout) {
-      if (!this.textContent || this.renderingDone) {
+      if (!this.textContentStream || this.renderingDone) {
         return;
       }
       this.cancel();
 
       this.textDivs = [];
       var textLayerFrag = document.createDocumentFragment();
       this.textLayerRenderTask = renderTextLayer({
-        textContent: this.textContent,
+        textContentStream: this.textContentStream,
         container: textLayerFrag,
         viewport: this.viewport,
         textDivs: this.textDivs,
         timeout,
         enhanceTextSelection: this.enhanceTextSelection,
       });
       this.textLayerRenderTask.promise.then(() => {
+        this.textContentItemsStr = this.textLayerRenderTask.textContentItemsStr;
```

Comment:
Isn't it better to follow the same pattern as is used with `this.textDivs` for this property as well?